test/test_math_ops.cpp

```diff
@@ -1614,6 +1614,28 @@ TEST(MathOps, Log2D) {
   opexecuter.RunTest();
 }  // end of test op Log
 
+TEST(MathOps, Log1p) {
+  Scope root = Scope::NewRootScope();
+  int dim1 = 2;
+  int dim2 = 3;
+  vector<float> vals = {0, 0.25, 0.5, 1, 5, 10};
+
+  Tensor A(DT_FLOAT, TensorShape({dim1, dim2}));
+
+  AssignInputValues(A, vals);
+
+  vector<int> static_input_indexes = {};
+  auto R = ops::Log1p(root, A);
+
+  vector<DataType> output_datatypes = {DT_FLOAT};
+
+  std::vector<Output> sess_run_fetchoutputs = {R};
+  OpExecuter opexecuter(root, "Log1p", static_input_indexes, output_datatypes,
+                        sess_run_fetchoutputs);
+
+  opexecuter.RunTest();
+}  // end of test op Log1p
+
```

Comment:
can u add a pytest as well: https://www.tensorflow.org/api_docs/python/tf/math/log1p
also please make sure using NGRAPH_TF_LOG_PLACEMENT that log1p cpp op is being created if u use the python api `tf.math.log1p`

Reply:
Also, can you please try to discover some TF python tests using this: https://github.com/tensorflow/ngraph-bridge/tree/master/test/python/tensorflow

things like these: https://github.com/tensorflow/tensorflow/blob/9716a8099eafc84c479d49b62e1eb2bbf18af33c/tensorflow/python/ops/math_ops_test.py#L567

Reply:
@sayantan-nervana - I added the TF test cases in the "test/python/tensorflow/python_tests_list.txt " file, did the make install and pip install tf_br whl,  but when I run the test_runner.py or the tf_unittest_runner.py, those test cases are output as INVALID. It seems to me that there are additional update needed when adding the TF's test cases.
Did I miss copying some files or install additional whl? Thanks

Reply:
copy the build_cmake/python/dist<whlfile> to ./build_cmake/artifacts

Reply:
Done. Thanks!